move/its/sources/coin_management.move

```diff
@@ -89,7 +89,7 @@ module its::coin_management {
     // === Protected Methods ===
 
     /// Takes the given amount of Coins from user. Returns the amount that the ITS is supposed to give on other chains.
-    public(package) fun take_balance<T>(self: &mut CoinManagement<T>, to_take: Balance<T>, clock: &Clock): u256 {
+    public (package) fun take_balance<T>(self: &mut CoinManagement<T>, to_take: Balance<T>, clock: &Clock): u256 {
```

Comment:
is this being done automatically by move formatter? Since `public(package) fun ...` seems to be the convention as per the move [book](https://move-book.com/move-basics/visibility.html#package-visibility)